core/src/scss/utilities/mixins/button/_button-primary.scss

```diff
@@ -9,14 +9,25 @@
 @mixin button-primary {
   @extend %button-base;
   @include padding(1rem 2rem 1.15rem);
-
   background-color: color(brand--bright);
   color: color(link--reverse);
 
+  &::after,
+  &::before {
+    background-color: color(link--reverse);
+    color: color(brand--bright);
+  }
+
   &:hover,
   &:focus {
     background-color: color(bg--reverse);
     color: color(link--reverse);
+
+    &::after,
+    &::before {
+      background-color: color(link--reverse);
+      color: color(bg--reverse);
+    }
   }
```

Comment:
@cjwest In the button scss files, for the ::after and ::before elements, I see that the background-color property changes the icon color of the svg icons, but what is the color property for here? When I toggle the color off in the inspector, it still renders the button with mail icon correctly
![Screen Shot 2021-09-10 at 12 45 33 AM](https://user-images.githubusercontent.com/42749717/132818932-6e36778f-410c-4ec6-b236-19c1f702c256.png)

Reply:
Hi @yvonnetangsu ,

I will remove the color.

Just so you don't think I'm crazy, I did have it in there if you use the `content: ""` attribute rather than the mask.

For example:
![Screen Shot 2021-09-22 at 11 21 42 AM](https://user-images.githubusercontent.com/284440/134400510-75903859-12ac-4dbd-8afe-38243ec9ab8e.png)



But then you need to change the background for the content to show. So, it would need extra css in any case.

Reply:
Ah I see what you're saying 😄